### PR TITLE
[Teamdmgrd]Fix "teamd cannot start when port channel was down before warm reboot" issue

### PR DESCRIPTION
**Pull request by shine4chen**
**What I did**
This PR is used to Fix issue #1165
Sometimes teamd could save the lacp-pdu on negotiation stage. The partner field of this pdu is empty and  can't be used to recover lacp mac-address.  So we add some validation here.

**Why I did it**

**How I verified it**
Build and test on lab

**Details if related**

### CODE REVIEW

**Comment by shine4chen** [comment]
retest this please

File: cfgmgr/teammgr.cpp [hunk 1]
```diff
@@ -492,13 +492,19 @@ task_process_status TeamMgr::addLag(const string &alias, int min_links, bool fal
...
 
             if (!memberfile.is_open())
                 continue;
 
             memberfile.seekg(partner_system_id_offset, std::ios::beg);
             memberfile.read(reinterpret_cast<char*>(mac_temp), ETHER_ADDR_LEN);
+
+            /* During negotiation stage partner info of pdu is empty , skip it */
+            if (memcmp(mac_temp, null_mac, ETHER_ADDR_LEN) == 0)
+		continue;
```
Comment (stepanblyschak) [inline cfgmgr/teammgr.cpp]: Please, replace tabs with spaces

Comment (shine4chen) [reply cfgmgr/teammgr.cpp]: done

**Comment by shine4chen** [comment]
retest this please

**Comment by shine4chen** [comment]
retest this please

**Comment by shine4chen** [comment]
retest this please

**Pull request merged**